examples/launch_hpob.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from syne_tune.blackbox_repository import (
+    load_blackbox,
+    BlackboxRepositoryBackend,
+)
+
+from syne_tune.backend.simulator_backend.simulator_callback import SimulatorCallback
+from syne_tune.optimizer.baselines import RandomSearch
+from syne_tune import StoppingCriterion, Tuner
+
+
```

Comment:
Can we add a brief comment here explain what this example does? Something like: 'This example shows how to run RandomSearch on Benchmarks from HPO-B by Pineda et al. We use a XGBoost surrogate mode to predict the performance of unobserved hyperparameter configurations.'